src/easyaccess.cpp

```diff
@@ -184,360 +186,328 @@ namespace Exiv2 {
 
     ExifData::const_iterator flashBias(const ExifData& ed)
     {
-        static const char* keys[] = {
-            "Exif.CanonSi.FlashBias",
-            "Exif.Panasonic.FlashBias",
-            "Exif.Olympus.FlashBias",
-            "Exif.OlympusCs.FlashExposureComp",
-            "Exif.Minolta.FlashExposureComp",
-            "Exif.SonyMinolta.FlashExposureComp",
-            "Exif.Sony1.FlashExposureComp",
-            "Exif.Sony2.FlashExposureComp"
-        };
-        return findMetadatum(ed, keys, EXV_COUNTOF(keys));
+        const auto keys = {"Exif.CanonSi.FlashBias",         "Exif.Panasonic.FlashBias",
```

Comment:
Two comments here:

1. I think the usage of `static` here was intentional and right. By removing it, this array is created each time we access the function instead of having a static instance during the application lifetime.

2. I do not like the usage of `auto` here. It makes more difficult for the code reader to guess the type. I would prefer to use `static const std::array<const char*,  N>`. 

The same applied for all the other definitions of the variables `keys`.

Reply:
So auto here does not evaluate to std::array. It evaluates to std::initializer_list.

As far as I can tell, the latter is more restricted in terms of what it can do.

Reply:
That's the thing, using `auto` is not straightforward to guess the type in this specific case :) . I think it would be better to use the exact type instead to avoid confusion (either std:array or std::initializer_list)